src/mpi/romio/adio/ad_quobytefs/ad_quobytefs.c

```diff
@@ -0,0 +1,116 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/**
+ *   Copyright 2019 Quobyte Inc. All rights reserved.
+ *   Author: alexey@quobyte.com
+ *   See COPYRIGHT notice in top-level directory.
+ */
+
+
+#include "ad_quobytefs.h"
+
+#include "adioi.h"
+#include <string.h>
+
+struct ADIOI_Fns_struct ADIO_QUOBYTEFS_operations = {
+    ADIOI_QUOBYTEFS_Open,       /* Open */
+    ADIOI_GEN_OpenColl, /* OpenColl */
+    ADIOI_QUOBYTEFS_ReadContig, /* ReadContig */
+    ADIOI_QUOBYTEFS_WriteContig,        /* WriteContig */
+    ADIOI_GEN_ReadStridedColl,  /* ReadStridedColl */
+    ADIOI_GEN_WriteStridedColl, /* WriteStridedColl */
+    ADIOI_GEN_SeekIndividual,   /* SeekIndividual */
+    ADIOI_QUOBYTEFS_Fcntl,      /* Fcntl */
+    ADIOI_GEN_SetInfo,  /* SetInfo */
+    ADIOI_GEN_ReadStrided,      /* ReadStrided */
+    ADIOI_GEN_WriteStrided,     /* WriteStrided */
+    ADIOI_QUOBYTEFS_Close,      /* Close */
+#ifdef ROMIO_HAVE_WORKING_AIO
+    ADIOI_QUOBYTEFS_IreadContig,        /* IreadContig */
+    ADIOI_QUOBYTEFS_IwriteContig,       /* IwriteContig */
+#else
+    ADIOI_FAKE_IreadContig,     /* IreadContig */
+    ADIOI_FAKE_IwriteContig,    /* IwriteContig */
+#endif
+    ADIOI_GEN_IODone,   /* ReadDone */
+    ADIOI_GEN_IODone,   /* WriteDone */
+    ADIOI_GEN_IOComplete,       /* ReadComplete */
+    ADIOI_GEN_IOComplete,       /* WriteComplete */
+    ADIOI_GEN_IreadStrided,     /* IreadStrided */
+    ADIOI_GEN_IwriteStrided,    /* IwriteStrided */
+    ADIOI_QUOBYTEFS_Flush,      /* Flush */
+    ADIOI_QUOBYTEFS_Resize,     /* Resize */
+    ADIOI_QUOBYTEFS_Delete,     /* Delete */
+    ADIOI_GEN_Feature,  /* Features */
+    "QUOBYTEFS:ROMIO driver for quobyte file system",
+    ADIOI_GEN_IreadStridedColl, /* IreadStridedColl */
+    ADIOI_GEN_IwriteStridedColl,        /* IwriteStridedColl */
+    ADIOI_QUOBYTEFS_SetLock     /* SetLock */
+};
+
+char *ADIOI_QUOBYTEFS_ExtractRegistry(const char *filename, int *error_code)
```

Comment:
We use underscore formats for function names, rather than camelcase.

Reply:
I used the same convention as in other interfaces e.g xfs:
https://github.com/pmodels/mpich/blob/e200df5cb224a196c4e61f64a6d044701313541a/src/mpi/romio/adio/ad_xfs/ad_xfs.c#L21

Reply:
Alright, let's leave it for now.  We'll go and clean up all of the function names at the same time.